### ISSUE TITLE
ColorVariable incorrectly reporting error

### ISSUE THREAD

**Issue by ryanmc2033** [brigade/scss-lint#482]
I have a color map that looks like this 
```
$green: #9ecc3b;
$white: #fff;
$red: #f00;

$colors: (
    'green': $green,
    'white': $white,
    'red': $red
);
```

and I am getting errors like this
```
assets/sass/base.scss:50 [W] ColorVariable: Color literals like `green` should only be used in variable declarations; they should be referred to via variable everywhere else.
assets/sass/base.scss:51 [W] ColorVariable: Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.
assets/sass/base.scss:52 [W] ColorVariable: Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.
```

**Comment by sds** [comment]
Which version of `scss-lint` are you using, @ryanmc2033?

Please upgrade to the latest version of `scss-lint`, as a similar issue was reported in #402 and has been fixed.
